net-misc/gupnp-tools/gupnp-tools-0.8.11.ebuild

```diff
@@ -30,3 +30,11 @@ DEPEND="${RDEPEND}
 	sys-devel/gettext
 	virtual/pkgconfig
 "
+
+src_prepare() {
+	# Fixes 598284 with upstream patch
+	# https://git.gnome.org/browse/gupnp-tools/commit/?id=e9ec9634207e4c2eea6d268ee29b57e687c1f178
+	epatch "${FILESDIR}"/fix-compiling-when-using-recent-gupnp-av.patch
+	gnome2_src_prepare
```

Comment:
consider whether `EAPI=6` bumping + `PATCHES` isn't easier than beating the dead `utils` horse